settings.gradle.kts

```diff
@@ -1,8 +1,34 @@
+@file:Suppress("UnstableApiUsage")
+
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
+
 pluginManagement {
     repositories {
-        //mavenLocal()
-        mavenCentral()
         gradlePluginPortal()
+        maven("https://repo.opencollab.dev/main/")
+        maven("https://jitpack.io") {
+            content {
+                includeGroupByRegex("com\\.github\\..*")
+            }
+        }
+
+        maven("https://maven.architectury.dev/")
+        maven("https://maven.neoforged.net/releases")
         maven("https://maven.fabricmc.net/")
     }
+
+    plugins {
+        id("net.kyori.blossom") version "1.2.0"
```

Comment:
can blossom be added as a plugin to the version catalogue so that the version is declared there

Reply:
is it possible to access the libs catalogue from the settings? didnt want to work for me when i tried it